pages/common/git-rm.md

```diff
@@ -0,0 +1,15 @@
+# git rm
+
+> Remove files from the working tree and from the index
+
+- Remove file and add its deletion to next commit
+
+`git rm {{file}}`
+
+- Remove file from all previous commits but keep it locally
+
+`git rm --cached {{file}}`
+
+- Remove directory
```

Comment:
I think this example should come after `git rm {{file}}`, before the `--cached` option.